server/webapp/WEB-INF/rails.new/webpack/views/shared/page_load_error.js.msx

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
any reason for this being a separate file? can we include this as part of `form_helpers`?

Reply:
Or the form helper can be split into separate components?